package-examples/hello-composite-pkg/hello-subpkg/README.md

```diff
@@ -0,0 +1,16 @@
+hello-subpkg
+==================================================
+
+# NAME
+
+  hello-subpkg
+
+# SYNOPSIS
+
+  kubectl apply --recursive -f hello-subpkg
```

Comment:
remove this as it doesn't make sense. @mortent is working on the issue for generating a useful README.

Reply:
Done.